.github/workflows/README.md

```diff
@@ -0,0 +1,67 @@
+# Toolbelt Workflow
+
+Goal: A mechanism that implements as many OpenSSF best practices using OpenSSF tools as possible in a way that is simple to use for project maintainers and easy for end users to check projects using this mechanism are following best practices and creating important security metadata (e.g. SBOMs, in-toto SLSA, etc.)
+
+Secondary Goals:
+
+- Highlight where there are gaps in OpenSSF and related practices, frameworks, and tools, or their adoption, especially in different ecosystems (e.g. Sigstore signatures are not supported in Maven)
+- Highlight when implementing these practices or adopting the tools is difficult
+Show examples following these practices and implementing the tools 
+
+Outcomes:
+
+- Maintainers should be able to use this mechanism to implement OpenSSF practices and adopt OpenSSF tooling, e.g. add SLSA to their build and turn on Scorecard.
+- Maintainers should be able to easily know where data is generated, e.g. where SBOMs, SLSA, SAST reports, etc. live so they can use it for later analysis and understanding supply chain risks.
```

Comment:
This is might be too complex for the first version of the workflow, but I think it would be great if the workflow incorporated some standard policy checking at certain stages. Maybe it's enough to even start with something basic like signature checking, but the idea is to further reduce extra analysis that maintainers have to do.

Reply:
Yes, I think that would be something on the roadmap. I was even thinking this could set up a basic in-toto layout.